grpc/core/src/main/java/io/helidon/grpc/core/GrpcTlsDescriptor.java

```diff
@@ -161,6 +166,7 @@ public Builder jdkSSL(boolean jdkSSL) {
          * @param tlsCert the path to client's certificate
          * @return this instance for fluent API
          */
+        @ConfiguredOption(key = "tls-cert")
```

Comment:
Explicit key is not needed here. Name `tlsCert` would be translated to `tls-cert` by the generator.